src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -425,6 +427,11 @@ private void injectRealIds(FeedbackResponseCommentAttributes responseComment) {
                 + "%" + responseIdParam[1] + "%" + responseIdParam[2];
     }
 
+    /**
+     * Removes existing entities of {@code dataBundle} in datastore.
+     * Removing {@link AdminEmailAttributes} should be avoided in production environment,
+     * as non-testing data may be removed.
```

Comment:
The part on "removing admin emails in production should be avoided" already includes implementation details. It should be a code comment instead of header comment. After all, users of the method `removeDataBundle` need not know in detail that such removal is to be avoided, right?